biom/table.py

```diff
@@ -271,7 +274,7 @@ def vlen_list_of_str_formatter(grp, header, md, compression):
         if m[header] is None:
             continue
         value = np.asarray(m[header])
-        data[i, :len(value)] = value
+        data[i, :len(value)] = [str(v).encode('utf8') for v in value]
```

Comment:
It's possible. May not have back tracked this appropriately re: gutter
conversation
On Oct 20, 2015 12:54 PM, "Evan Bolyen" <notifications@github.com> wrote:

> In biom/table.py
> <https://github.com/biocore/biom-format/pull/661#discussion_r42537893>:
>
> > @@ -271,7 +274,7 @@ def vlen_list_of_str_formatter(grp, header, md, compression):
> >          if m[header] is None:
> >              continue
> >          value = np.asarray(m[header])
> > -        data[i, :len(value)] = value
> > +        data[i, :len(value)] = [str(v).encode('utf8') for v in value]
>
> Just to check v would never be a bytestring correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/biom-format/pull/661/files#r42537893>.
>